WORKSPACE

```diff
@@ -38,6 +38,30 @@ load("//mixer:adapter_author_deps.bzl", "mixer_adapter_repositories")
 
 mixer_adapter_repositories()
 
+go_repository(
```

Comment:
Do we still need upgrading WORKSPACE ?

Reply:
Not sure. Short term it was easy to update while we migrate away from bazel as authoritative build system. Longer term it will be a pain to update multiple sync files. Fortunately `gazelle` recently learned how to auto-update WORKSPACE from `Gopkg.lock`. See https://github.com/bazelbuild/bazel-gazelle#update-repos. We could remove WORKSPACE (or provide a minimal / stripped down version) and auto-generate BUILD and WORKSPACE.